MarkupsToModel/Logic/vtkSlicerMarkupsToModelLogic.cxx

```diff
@@ -276,135 +282,147 @@ void vtkSlicerMarkupsToModelLogic::UpdateSelectionNode( vtkMRMLMarkupsToModelNod
 }
 
 //------------------------------------------------------------------------------
-void vtkSlicerMarkupsToModelLogic::UpdateOutputCloseSurfaceModel(vtkMRMLMarkupsToModelNode* markupsToModelModuleNode, vtkPolyData* output)
+void vtkSlicerMarkupsToModelLogic::UpdateOutputCloseSurfaceModel( vtkMRMLMarkupsToModelNode* markupsToModelModuleNode, vtkPolyData* output )
 {
   vtkMRMLMarkupsFiducialNode* markups = markupsToModelModuleNode->GetMarkupsNode();
-  if(markups == NULL)
+  if ( markups == NULL )
   {
     return;
   }
   int numberOfMarkups = markups->GetNumberOfFiducials();
 
-  if (numberOfMarkups == 0)
+  if ( numberOfMarkups == 0 )
   {
     return;
```

Comment:
By "clean the displayed model" I meant remove all points in the model (to not show anything).
When there are no input markups then we should make sure we do not display a surface (previously there could have been markups that generated a surface).